cmd/ethrex/initializers.rs

```diff
@@ -93,16 +93,19 @@ pub fn open_store(data_dir: &str) -> Store {
 #[cfg(feature = "l2")]
 pub async fn init_rollup_store(data_dir: &str) -> StoreRollup {
     cfg_if::cfg_if! {
-        if #[cfg(feature = "rollup_storage_redb")] {
+        if #[cfg(feature = "rollup_storage_sql")] {
```

Comment:
Shouldn't this be under the l2 flag? If we want the sql db to debug things, disregard this comment.